src/nwb_datajoint/spikesorting/spikesorting_curation.py

```diff
@@ -66,18 +74,28 @@ def insert_curation(sorting_key: dict, parent_curation_id: str = 'original',
             metrics = {}
 
         # generate a unique ID
```

Comment:
this generates curation number not unique ID

Reply:
Agreed and I'll fix this